src/components/header/Header.component.tsx

```diff
@@ -32,68 +39,78 @@ const Header: React.FC<HeaderProps> = ({ onShowAbout, onShowProducts }) => (
                                         strokeLinejoin="round"
                                     />
                                 </svg>
-                            </svg>
-                        </button>
-                        <img className={styles.header__theme_icon} src={headerThemeElement} alt="Element theme" />
-                        <button className={styles.header__theme_button_dark}>
-                            <svg width="30" height="34" viewBox="0 0 30 34" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            </button>
+                            <img className={styles.header__theme_icon} src={headerThemeElement} alt="Element theme" />
+                            <button
+                                className={styles.header__theme_button_dark}
+                                onClick={showTheme}
+                                style={{ color: theme === 'dark' ? '#FFF' : '#656565' }}
```

Comment:
You have "className={${theme}-theme}". Try to change styles inside css class.

style={{ color: theme === 'dark' ? '#FFF' : '#656565' }} push it to className={styles.header__theme_button_dark}